test/unit-test-helpers/mock_ima.js

```diff
@@ -146,10 +146,6 @@ google =
                 amCallbacks[event] = callback;
               };
               this.start = function() {
-                //Use the canPublishEvents flag to ensure that start
-                //gets called, otherwise publish events will fail,
-                //failing unit tests
-                canPublishEvents = true;
```

Comment:
We no longer need this since we no longer have setAutoPlayAdBreaks set to false.